### ISSUE TITLE
question: what is causing mystical agriculture crops to grow so fast in garden cloches?

### ISSUE THREAD

**Issue by Quezler** [NillerMedDild/Enigmatica2Expert#1932]
**Modpack version**
1.82a

**Issue**
not being able to stop thinking about what's causing it

**Reasoning as to why ask it as an issue**
- seemingly no dedicated e2e channel on discord, and e2/support has a lot of messages so it can get brried
- asking it on github practically guarantees the person that knows about it will see it
- why an issue out of all things? well unlike [create together](https://github.com/NillerMedDild/CreateTogether/discussions/categories/q-a) this repository doesn't have discussions enabled, which would have been a more appropriate place to ask this

**The full question**
i've just been really curious what exactly is causing the mystical agriculture crops to grow as quickly as they do in garden cloches in this pack, normally the crops are slower than slow, but having them in garden cloches grows them on steroids.

i would really like to know which mod/config/bug is responsible for this behavior, since in other packs with those 2 mods banged together their speed is a slow as you'd expect, and this thought has been in my mind ever since my first e2e playthrough 1-2 years ago (now on my 3rd or higher?)

(i did try to search inside the configs/repo then & now, but never really noticed anything obvious enough to notice)

**So in a nutshell**
- just a quick description of how exactly this was achieved in this pack, please
- possibly open "discussions" on this repo to prevent further misuse of issues

**Comment by NillerMedDild** [comment]
Immersive Engineering configs are causing the Garden Cloche to be much faster if you give it fertilizer, at the cost of way more power.
I've opened discussions :)
